pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.isort]
+profile = 'black'
+
+[tool.black]
+line-length = 120
+target-version = ['py38']
+include = '\.pyi?$'
+extend-exclude = '''
+/(
+    \.git
+  | \.mypy_cache
+  | \.venv
+  | migrations
+)/
+'''
+
+[tool.mypy]
+strict = false
+python_version = "3.8"
+ignore_missing_imports = true
+warn_return_any = true
+exclude = ['venv', '.venv', 'migrations']
```

Comment:
By the way, may be we will add this hook as well?
![image](https://github.com/ScilifelabDataCentre/stackn/assets/8545082/e48b4722-5e44-4235-833c-7c138eb0e87b)
https://pre-commit.com/hooks.html

Reply:
Yes

Reply:
I will add it to the list of future improvements.

Reply:
I don't see a reason, why not add that now. I remember your concerns regrading potential explosion of changes, but It's not that big of a change :)

But if you insist, sure, let's do that later

Reply:
I have now added this hook.